src/main/java/uk/gov/hmcts/reform/amapi/controllers/AccessManagementController.java

```diff
@@ -48,7 +48,7 @@
                                      @RequestHeader(name = "Caller", defaultValue = "Anonymous") String caller) {
```

Comment:
API end points needs to be changed as they are documented in confluence

Reply:
done